pilot/cmd/pilot-agent/options/agent.go

```diff
@@ -44,7 +44,6 @@ func NewAgentOptions(proxy *model.Proxy, cfg *meshconfig.ProxyConfig) *istioagen
 		ServiceNode:                 proxy.ServiceNode(),
 		EnvoyStatusPort:             envoyStatusPortEnv,
 		EnvoyPrometheusPort:         envoyPrometheusPortEnv,
-		MinimumDrainDuration:        minimumDrainDurationEnv,
```

Comment:
Please see here on why it was added https://github.com/istio/istio/pull/35059#pullrequestreview-758922774